kikit/panelize_ui_sections.py

```diff
@@ -297,8 +309,16 @@ def never():
         "Reference renaming pattern"),
     "baketext": SBool(
         always(),
-        "Substitute variables in text elements"
-    ),
+        "Substitute variables in text elements"),
+    "destination": SVec(
+        always(),
+        "Destination of source board"),
```

Comment:
As pointed out, I believe the destination shouldn't be present for the grid and plugin types of layout (as pointed out above).